pkg/landscaper/gardenlet/apis/imports/types.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package imports
+
+import (
+	"encoding/json"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	gardenletconfigv1alpha1 "github.com/gardener/gardener/pkg/gardenlet/apis/config/v1alpha1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// LandscaperGardenletImport defines the landscaper import for the Gardenlet.
+// structure defined in Blueprint
+type LandscaperGardenletImport struct {
+	metav1.TypeMeta
+	// RuntimeCluster is the landscaper target containing the kubeconfig for the cluster
+	// where the Gardenlet should be deployed.
+	// This is the Kubernetes cluster targeted as Seed (via in-cluster mounted service account token),
+	// if not otherwise specified in `.componentConfiguration.seedClientConnection.kubeconfig`.
+	RuntimeCluster Target
+	// GardenCluster is the landscaper target containing the kubeconfig for the
+	// Garden cluster (having Gardener resource groups!)
+	GardenCluster Target
+	// ImageVectorOverwrite contains the image vector override
+	ImageVectorOverwrite *string
+	// ImageVectorOverwrite contains the image vector override for components deployed by the gardenlet
+	ComponentImageVectorOverwrites *runtime.RawExtension
```

Comment:
missed that one in the first run.
It is the same as the `ImageVectorOverwrite` ?

Reply:
regarding the spelling? Because it is plural?

Reply:
I guess the question is about the type. Why is `ImageVectorOverwrite` a string ptr and `ComponentImageVectorOverwrites` a raw extension ptr?